src/gui/gui_msgs.h

```diff
@@ -37,35 +37,35 @@ constexpr char help_msg[] =
 
 List of available options:
 
-  --conf <configfile>      Start with the options specified in <configfile>.
-                           Multiple configfiles can be specified.
+  --conf <config_file>     Start with the options specified in <config_file>.
+                           Multiple configuration files can be specified.
                            Example: --conf conf1.conf --conf conf2.conf
 
-  --printconf              Print the location of the default configuration file.
-                           If it does not exist, create it.
+  --printconf              Print the location of the primary configuration file.
 
-  --editconf               Open the default configuration file in a text editor.
+  --editconf               Open the primary configuration file in a text editor.
 
-  --eraseconf              Delete the default configuration file.
+  --eraseconf              Delete the primary configuration file.
 
-  --noprimaryconf          Don't read settings from the default configuration file
-                           located in your user folder.
+  --noprimaryconf          Don't load the primary configuration file, and don't
+                           write it if it doesn't exist.
```

Comment:
Suggest:

`Don't load or create the the primary configuration file.`

Reply:
Haha, generally I'm good at not overcomplicating sentences, but yeah, this is a lot simpler 😄